src/examples/createSignatureOrder/sealsPageTemplate.cs

```diff
@@ -0,0 +1,33 @@
+using Criipto.Signatures;
+using Criipto.Signatures.Models;
+
+using (var client = new CriiptoSignaturesClient("{YOUR_CRIIPTO_CLIENT_ID}", "{YOUR_CRIIPTO_CLIENT_SECRET}")) {
+    // Setup document input
+    var documents = new List<DocumentInput>{
+        new DocumentInput {
+            pdf = new PadesDocumentInput {
+                title = "Example PDF",
+                blob = new Byte[64], // Should be the bytes of a PDF file
+                storageMode = DocumentStorageMode.Temporary,
+
+            sealsPageTemplate: {
+              blob: new Byte[64], // Should be the bytes of a PDF file
+              area: { x1: 33, x2: 561, y1: 388, y2: 820 },
+              expectedRows: 4,
+              expectedColumns: 2,
+            },
+            }
```

Comment:
There's an odd number of closing brackets `}`

Reply:
Thanks! I don't have any syntax checks in my VS Code for these files, sorry.